contrib/enfuse_pro_2.lua

```diff
@@ -0,0 +1,1050 @@
+--[[Enfuse professional plugin for darktable 2.2.X and 2.4.X
+
+  copyright (c) 2017, 2018  Holger Klemm (Original Linux-only version)
+  
+  darktable is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+  
+  darktable is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+  
+  You should have received a copy of the GNU General Public License
+  along with darktable.  If not, see <http://www.gnu.org/licenses/>.
+]]
+
+--[[ Enfuse Pro 2 - cross-platform compatible
+Modified by: Kevin Ertel
+   
+ADDITIONAL SOFTWARE NEEDED FOR THIS SCRIPT
+* align_image_stack
+* enfuse
+* exiftool
+
+USAGE
+* require this file from your main luarc config file.
+
+This plugin will add the new export modul "fusion to DRI or DFF image".
+]]
+
+local dt = require "darktable"
+local df = require "lib/dtutils.file"
+local gettext = dt.gettext
+
+-- works with LUA API version 4.0.0 and 5.0.0
+dt.configuration.check_version(...,{4,0,0},{5,0,0})
```

Comment:
You're using dt.print_log() so the compatible api version is only 5.0.0

Reply:
Didn't know that, great! Will remove 4.0.0 from the list.